docs/converters/tabular_converter.md

```diff
@@ -130,15 +128,12 @@ You can use the following `column` definitions:
     )
     ```
 ## Units
-Power Grid Model uses SI units (e.g. "W" for Watts),
-but source data may be supplied in different units (e.g. "MW" for Mega Watts).
-If units are supplied in the tabular data,
-the pandas DataFrame storing the data is expected to have `MultiIndexes` for the columns.
-For our application, a `MultiIndex` can be interpreted as a tuple; the first element is the column name, the second 
-element is the column unit. For example: `("C0", "µF")`.
+Power Grid Model uses SI units (e.g., "W" for Watts), but source data may be supplied in different units (e.g., "MW" for Mega Watts).
+If units are supplied in the tabular data, the data stored using pandas DataFrame is expected to have `MultiIndexes` for columns.
+For our application, a `MultiIndex` can be interpreted as a tuple; the first element is the column name, the second element is the column unit. For example: `("C0", "µF")`.
 
-If a unit is supplied, it should be defined in the units section of the mapping.
-Undefined units are not allowed to prevent errors.
+If a unit is supplied, it should be defined in the unit section of the mapping.
+Undefined units are not allowed.
```

Comment:
This `Undefined units` could use some clarification; are they units that are not in a certain list? or are they simply missing units?

Reply:
good point. my guess is "using a unit in the data file that is not listed in the mapping file" but not sure. let's investigate a little bit more

Reply:
@petersalemink95 Perhaps you could give us some insight here?

Reply:
Units can be supplied for some column. If that is the case, the meaning of the unit should be supplied in the units section of the mapping file. E.g. kW -> 1000

See https://github.com/PowerGridModel/power-grid-model-io/blob/main/src/power_grid_model_io/config/excel/vision_en.yaml to have a better understanding of the units section

Reply:
Resolved in the latest commit.